app/views/homes/show.html.erb

```diff
@@ -1,25 +1,31 @@
-<%= simple_form_for TelevisionShow.new do |form| %>
-  <%= form.input :title %>
-  <%= form.button :submit %>
-<% end %>
+<main>
+  <h2>Recommendations</h2>
+  <%= simple_form_for TelevisionShow.new do |form| %>
+    <%= form.input :title, label: false, placeholder: "The Wire" %>
+    <%= form.button :submit %>
+  <% end %>
 
-<% if current_user.following.empty? %>
-  <div class="no-followers">
-    <p><%= t(".not_following_anyone") %></p>
-    <ul>
-      <% current_user.users_to_follow.each do |user| %>
-        <li><%= link_to user.username, user %></li>
-      <% end %>
-    </ul>
-  </div>
-<% end %>
+  <ul>
+    <% @recommended_items.each do |recommended_item| %>
+      <li>
+        <strong>
+          <%= recommended_item.television_show_title %>
+        </strong>
+        &mdash; <%= link_to recommended_item.username, recommended_item.user %>
+      </li>
+    <% end %>
+  </ul>
+</main>
 
-<h2>Recommendations</h2>
-<ul>
-  <% @recommended_items.each do |recommended_item| %>
-    <li>
-      <%= recommended_item.television_show_title %>
-      &mdash; <%= link_to recommended_item.username, recommended_item.user %>
-    </li>
+<aside>
+  <% if current_user.following.empty? %>
```

Comment:
Maybe this should be in a different PR but what if this was just a generic "Recommended People To Follow" list, which was somehow _more prominent_ when there is no one? Or are those two separate things?